android/engine/src/test/java/org/smartregister/fhircore/engine/configuration/ConfigurationRegistryTest.kt

```diff
@@ -211,6 +212,7 @@ class ConfigurationRegistryTest : RobolectricTest() {
   }
 
   @Test
+  @Ignore("Fix this flaky test")
```

Comment:
I added an issue for this, https://github.com/opensrp/fhircore/issues/1423

Reply:
Noted